src/content/deployment/subctl/_index.en.md

```diff
@@ -95,7 +95,7 @@ deployment.
 <!-- markdownlint-disable line-length -->
 | Flag                               | Description
 |:-----------------------------------|:----------------------------------------------------------------------------|
-| `--cable-driver` `<string>`        | Cable driver implementation (defaults to strongSwan IPsec)
+| `--cable-driver` `<string>`        | Cable driver implementation. Available options are `strongswan` (default) and `libreswan`
```

Comment:
```suggestion
| `--cable-driver` `<string>`        | Cable driver implementation. Available options are `strongswan` (default), `libreswan` and `wireguard`.
```

Reply:
Good suggestion, but let's make sure we add it as a different commit or update the commit message of the proposed commit. I'd also recommend a serial comma.

Reply:
I thought about this while submitting the change. Are we OK with documenting WireGuard in here? Personally, I don't have much experience with it and I am not sure if there are other prerequisites we need to document /cc @mangelajo

Reply:
There are other prerequisites for WireGuard, see: https://github.com/submariner-io/submariner/blob/master/.github/workflows/e2e.yml#L31

Reply:
OK, let me change the commit message and PR description. I will also submit a separate PR for the WireGuard prerequisites.

Reply:
We're wording it as "_Available options are..._" which implies all of them so I think wireguard needs to be included. We already document all of them elsewhere so perhaps we should word it as: "_Available options include `strongswan` (default) and `libreswan`. See the complete list  of options [here](../architecture/gateway-engine)_".